### PR TITLE
express-intro- Connie Chin

### PR DESCRIPTION
**Pull request by connie-chin**
Screenshot 2024-04-04 at 11 44 00 AM

### CODE REVIEW

File: express-intro/server.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+app.listen(8080, () => {
+  console.log('Express server listening on port 8080');
+});
+
+app.use((req, res, next) => {
+  console.log('Hello, World!');
+  console.log('The date is', new Date());
+  next();
+});
```
Comment (BrettAlbright) [inline express-intro/server.ts:9]: Hey Connie, just make sure to place this code before the `app.listen` call

app.listen, should be the last app method in the file when you build a server

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
one small update!

**Review by Zhiwutian — APPROVED** [review]
👍🏼

File: express-intro/server.ts (hunk 1 above)
Comment (connie-chin) [reply express-intro/server.ts:9]: got it! fixed 
Screenshot 2024-04-04 at 11 55 45 AM

**Pull request merged**